patternfly/patternfly-elements
# feat: [pfe-cta] Add disabled and context styles to CTA component, & clean up light dom

## Add disabled and context styles to CTA component

<img width="2478" alt="Screen Shot 2019-11-22 at 3 08 09 PM" src="https://user-images.githubusercontent.com/1840295/69457019-1af8d200-0d3a-11ea-9d83-4e4c9fa2fc0d.png">

* Link(s) to demo pages where this element can be viewed:
https://deploy-preview-630--happy-galileo-ea79c4.netlify.com/elements/pfe-cta/demo/
---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Add `aria-disabled="true"` styles
* Update existing variants to react to `on` context
* Remove extraneous Light DOM styles by adding a `pfe-cta--inner` element inside the wrapper
* Organized the sass file so that like-events were near each other and reduce duplication

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the demo page and confirm that styles in different theme contexts update based on those contexts
2. Open the demo page and confirm that the disabled CTA is styled appropriately

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
